### ISSUE TITLE
[6.x] Add template syncing to the cert and awarded cert post tables 

### ISSUE THREAD

**Issue by thomasplevy** [gocodebox/lifterlms#1933]
As a post row action

I think that a bulk action *should* also be possible (spawning multiple processors) but if the bulk action isn't possible then that's fine, I think.

This will be dependent on #1931 as currently achievments don't have any sync functionality

**Comment by thomasplevy** [comment]
@pondermatic I don't believe you tackled this in #1946 

I'm going to keep this on you but I'd like all the current code on that PR solidified and approved before adding this. This can be tackled as a separate PR.
